library/std/src/net/ip.rs

```diff
@@ -212,7 +212,7 @@ impl IpAddr {
     /// future, as new IETF RFCs are published.
     ///
     /// [changes]: super#stability-guarantees-for-ietf-defined-behavior
-    #[rustc_const_unstable(feature = "const_ip", issue = "76205")]
+    #[rustc_const_stable(feature = "const_ip", since = "1.50.0")]
```

Comment:
The 1.49 release is in two days, so this should be 1.51 instead. (nightly already shows a 1.51 version number: https://github.com/rust-lang/rust/pull/80366)

Reply:
Oh wait sorry, this was from a merge commit. Ignore me.

Reply:
lol